resultstore-search-api/db_client.py

```diff
@@ -0,0 +1,78 @@
+import sys
+import psycopg2
+import logging
+from psycopg2.extras import DictCursor
```

Comment:
This is actually what I was trying to avoid.
Using an adapter means that you are probably going to be writing direct SQL to send along to Postgres.

My recommendation is to consider shifting the backend onto Django and using Django ORM to interact with Postgres.
Alternatively, you can just import the ORM portion of Django to do database manipulation and interaction.